unfccc_di_api/unfccc_di_api.py

```diff
@@ -72,6 +74,76 @@ def __init__(
         KeyError.__init__(self, f"Query returned no data for: {query}")
 
 
+class ZenodoReader:
+    """Provides simplified unified access to the data provided by the Flexible Query
+    API of the UNFCCC data access, via the dataset stored at zenodo.
+
+    Essentially gives you the same API as the UNFCCCApiReader, but without complications
+    due to the protection measures of the DI API. The advantage of using the
+    ZenodoReader is that it works reliably without special measures, the disadvantage
+    is that the data might be a bit older.
+
+    Attributes
+    ----------
+    parties : pandas.DataFrame
+        All parties, with their ID, code, and full name.
+    """
+
+    def __init__(
+        self,
+        *,
+        url: str = "doi:10.5281/zenodo.7432088/data-2022-12-13.zip",
+        known_hash: str = "md5:e7f660f88425078c75093d19841e5815",
+    ):
```

Comment:
How can this deal with updated data on zenodo? Do we hardcode the hash and filename here?

Reply:
Yeah, that was my plan for now. Data updates on zenodo are a manual process anyway, and adding a new release of this package here to the list of things to do is not too bad, adding probably only about 20% overhead. If we find out we're releasing new data on zenodo often (at least, more often than currently, which was about twice a year), we can automate further.